src/components/batch-selection/batch-selection.test.js

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import { mount } from "@cypress/react";
+import BatchSelection from "./batch-selection.component";
+import IconButton from "../icon-button/icon-button.component";
+import { positionOfElement } from "../../../cypress/support/helper";
+import Icon from "../icon/icon.component";
+
+import {
+  batchSelectionCounter,
+  batchSelectionComponent,
+  batchSelectionButtonsByPosition,
+} from "../../../cypress/locators/batch-selection/index";
+
+context("Tests for BatchSelection component", () => {
+  describe("check BatchSelection component count", () => {
```

Comment:
in this `describe` we are checking the work of all component instead of `counter` only